### PR TITLE
Exports the cluster's name

### PR DESCRIPTION
**Pull request by raaizik**
[RHSTOR-5500](https://issues.redhat.com/browse/RHSTOR-5500)
Expands the `StorageConsumer`'s status to include the cluster's name which will then be exposed to the GUI back-end service.

### CODE REVIEW

File: api/v1alpha1/storageconsumer_types.go [hunk 1]
```diff
@@ -76,6 +76,8 @@ type ClientStatus struct {
 	ClusterID string `json:"clusterId"`
 	// NamespacedName is the name and namespace of the StorageClient
 	NamespacedName string `json:"namespacedName"`
+	// ClusterName is the name of the openshift cluster
+	ClusterName string `json:"clusterName"`
```
Comment (nb-ohad) [inline api/v1alpha1/storageconsumer_types.go:80]: Can we move that up under cluster ID?

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: services/provider/interfaces/interfaces.go [hunk 2]
```diff
@@ -8,11 +8,13 @@ type StorageClientStatus interface {
 	GetOperatorVersion() string
 	GetClusterID() string
 	GetNamespacedName() string
+	GetClusterName() string
```
Comment (nb-ohad) [inline services/provider/interfaces/interfaces.go:11]:
```suggestion
	GetClusterID() string
	GetClusterName() string
	GetNamespacedName() string
```

File: services/provider/interfaces/interfaces.go [hunk 3]
```diff
@@ -8,11 +8,13 @@ type StorageClientStatus interface {
...
 
 	SetPlatformVersion(string) StorageClientStatus
 	SetOperatorVersion(string) StorageClientStatus
 	SetClusterID(string) StorageClientStatus
 	SetNamespacedName(string) StorageClientStatus
+	SetClusterName(string) StorageClientStatus
```
Comment (nb-ohad) [inline services/provider/interfaces/interfaces.go:17]:
```suggestion
	SetClusterID(string) StorageClientStatus
	SetClusterName(string) StorageClientStatus
	SetNamespacedName(string) StorageClientStatus
```

**Comment by rchikatw** [comment]
changes look ok to me.

**Review by leelavg — APPROVED** [review]
please expand the PR summary and description, rest lg.

**Pull request merged**